board/safety/safety_toyota.h

```diff
@@ -180,7 +185,12 @@ static int toyota_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
           tx = 0;
         }
       }
-      bool violation = max_limit_check(desired_accel, TOYOTA_MAX_ACCEL, TOYOTA_MIN_ACCEL);
+      bool violation = false;
+      if (!(unsafe_mode & UNSAFE_RAISE_LONGITUDINAL_LIMITS_TO_ISO_MAX)) {
```

Comment:
Perhaps inline ifs here on max_limit_check about which limit to use? No need to have a code branch.